projects/core/directives/hint/hint.component.ts

```diff
@@ -42,12 +42,7 @@ const GAP = 4;
 
 @Component({
     selector: 'tui-hint',
-    template: `
-        <span
-            *polymorpheusOutlet="content as text; context: context"
-            [innerHTML]="text"
-        ></span>
-    `,
```

Comment:
Dubious rule. I think it's ok to use template like this in decorator

Reply:
I agree, I think we can just rely on our own judgement whether or not we should move it to a separate file.